sources/packages/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -36,4 +56,145 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
       })
       .getMany();
   }
+
+  /**
+   * Save scholastic standing and create new assessment
+   * for an application.
+   * @param applicationId application id.
+   * @param scholasticStandingData scholastic standing data to be saved
+   */
+  async saveScholasticStandingCreateReassessment(
+    applicationId: number,
+    userId: number,
```

Comment:
Missing comment for the userId.
If the user is specific for auditing, I have been adding comment as below:
```typescript
* @param auditUserId user that should be considered the one that is
* causing the changes.
```